include/dra.h

```diff
@@ -459,7 +492,7 @@ extern s32 g_playerLevel;
 extern s32 g_playerExp;
 extern s32 g_playerGold;
 extern s32 g_killCount;
-extern s32 g_playerHeart;
+extern playerHeart g_playerHeart[];
```

Comment:
`g_playerHeartMax` (0x80097BAC) comes right after `g_playerHeart` (0x80097BA8). So either  `g_playerHeart` is really a `s32` or we need to replace `g_playerHeartMax` with `g_playerHeart->unk4`. On top of that the fields of `struct playerHeart` might be renamed as `current` and `max` if we decide to pursue that route.